CustomerOrganizer/CustomerOrganizer/MainWindow.py

```diff
@@ -1,7 +1,31 @@
 import tkinter
+from tkinter import *
+import FileIO
+from FileIO import *
 
-class Application(tkinter.Frame):
+class MainWindow(tkinter.Tk):
     def __init__(self, master = None):
         super().__init__(master)
         self.master = master
-        self.pack()
\ No newline at end of file
+        self.toolBar()
+
+        self.customer = FileIO.FileIO()
+
+    def donothing():
+        '''do nothing at all'''
+
+    def toolBar(self):
+        menubar = Menu(self)
+
+        filemenu = Menu(menubar, tearoff = 0)
+        filemenu.add_command(label = "New Customer", command = self.donothing)
+        filemenu.add_command(label = "Open Customer", command = self.donothing)
+        filemenu.add_command(label = "Save Customer", command = self.customer.saveFile)
```

Comment:
I'm getting an unhandled exception error here "'_tkinter.tkapp' object has no attribute 'customer'".

Try initializing `self.customer` before calling `self.toolBar()`

Reply:
that did work, thanks